### PR TITLE
Web Player install links

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Add links to install instructions for Airtame and Zoom Rooms.

Note: Pending link to "All Other Hardware", so it is hidden for now.

## Motivation and Context
Unlimited Plans and WebPlayer epic

## How Has This Been Tested?
Locally and on [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/partials/displays/download-player-modal.html [hunk 1]
```diff
@@ -5,58 +5,48 @@
...
+    <p class="mt-1">
+      <a class="madero-link" id="downloadAirtame" href="https://help.risevision.com/hc/en-us/articles/360061086311" target="_blank">
+        Airtame
+      </a>
+      <br/>
+      <a class="madero-link" id="downloadZoomRooms" href="https://help.risevision.com/hc/en-us/articles/360062378091" target="_blank">
+        Zoom Rooms
+      </a>
+      <br/>
+      <a ng-if="false" class="madero-link" id="downloadUbuntu32" href="https://help.risevision.com/hc/en-us/articles/360060814832" target="_blank">
```
Comment (alex-deaconu) [inline src/partials/displays/download-player-modal.html:46]: @ezequielc this link's `id` is a duplicate of the Ubuntu 32 above. Thanks.

**Review by alex-deaconu — APPROVED** [review]
👍

File: src/partials/displays/download-player-modal.html (hunk 1 above)
Comment (ezequielc) [reply src/partials/displays/download-player-modal.html:46]: @alex-deaconu Good catch, I will update it once I have the article link. It is coming by end of the day, so I will hold on releasing this. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I got a placeholder link to the article and added in the last commit. I will wait for Patrick to finalize the article before releasing this. Thanks

**Pull request merged**